### ISSUE TITLE
next header/footer syling in ODF->pdf converter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#102]
```
The test file CV.odt is incorrectly converted to pdf.

Header on the first page is different than on subsequent pages. But converter 
put header from the first page in the whole document.

It seems that attribute style:next-style-name="some new style" is not taken 
into account. Upon next page master style should be changed (with 
header/footer).
```

Original issue reported on code.google.com by `abe...@gmail.com` on 5 Apr 2012 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.9.8
```

Original comment by `abe...@gmail.com` on 18 May 2012 at 11:20
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 124 has been merged into this issue.
```

Original comment by `abe...@gmail.com` on 7 Aug 2012 at 11:28
